src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -268,40 +311,43 @@ export const AssetPriceDetails = ({ asset, collection }: AssetPriceDetailsProps)
     }
   }, [asset, itemsInBag])
 
-  const isOwner = asset.owner ? account?.toLowerCase() === asset.owner?.address?.toLowerCase() : false
-
-  if (isOwner) {
-    return <OwnerContainer asset={asset} />
+  const shareTweet = () => {
+    window.open(
+      `https://twitter.com/intent/tweet?text=Check%20out%20${
+        asset.name ?? `${asset.collectionName} #${asset.tokenId}`
+      }%20(${asset.collectionName})%20https://app.uniswap.org/%23/nfts/asset/${asset.address}/${
+        asset.tokenId
+      }%20via%20@uniswap`,
+      'newwindow',
+      `left=${(window.screen.width - TWITTER_WIDTH) / 2}, top=${
+        (window.screen.height - TWITTER_HEIGHT) / 2
+      }, width=${TWITTER_WIDTH}, height=${TWITTER_HEIGHT}`
+    )
   }
 
+  const isOwner = asset.owner ? account?.toLowerCase() === asset.owner?.address?.toLowerCase() : false
+
   return (
     <Container>
-      <OwnerInformationContainer>
-        <OwnerText
-          target="_blank"
-          href={`https://etherscan.io/address/${asset.owner.address}`}
-          rel="noopener noreferrer"
-        >
-          {asset.tokenType === 'ERC1155' ? (
-            ''
-          ) : (
-            <span> Seller: {isOwner ? 'you' : asset.owner.address && shortenAddress(asset.owner.address, 2, 4)}</span>
-          )}
-        </OwnerText>
-        <UploadLink
-          onClick={() => {
-            setCopied(window.location.href)
-          }}
-          target="_blank"
-        >
-          <Upload size={20} strokeWidth={2} />
-        </UploadLink>
-      </OwnerInformationContainer>
-
-      {cheapestOrder && asset.priceInfo ? (
+      <AssetInfoContainer>
+        <CollectionNameContainer>
+          <DefaultLink to={`/nfts/collection/${asset.address}`}>
+            <CollectionHeader>
+              {collection.collectionName} {collection.isVerified && <VerifiedIcon />}
+            </CollectionHeader>
+          </DefaultLink>
+          <UploadLink onClick={shareTweet} target="_blank">
```

Comment:
want to check to see if they want to update the icon? Feels weird to use an upload icon to go to a different page, let alone twitter

Reply:
its weird and they do